packages/web-console/src/scenes/Notifications/Notification/SuccessNotification/index.tsx

```diff
@@ -41,22 +48,61 @@ const ZapIcon = styled(Zap)`
   color: ${color("draculaYellow")};
 `
 
+const CopyIcon = styled(Copy)`
+flex-shrink: 0;
+`
+
 export const SuccessNotification = (props: NotificationShape) => {
-  const { createdAt, content, sideContent, jitCompiled } = props
+  const { createdAt, content, sideContent, jitCompiled, request, result } = props
+  const handleCopyToClipboard = useCallback(() => {
+    let clipboardText = `[${format("HH:mm:ss", createdAt)}] ${request?.query}\n`
+    if (result?.timings) {
+      try {
+        const { count, compiler, fetch, execute } = result.timings;
+        clipboardText += `✔ Success: ${result.count?.toLocaleString()} rows in ${formatTiming(fetch)}\n`;
+        clipboardText += new AsciiTable()
+          .setHeading('Execute', 'Network', 'Total', 'Count', 'Compile')
+          .addRow(
+            formatTiming(execute),
+            formatTiming(fetch - execute),
+            formatTiming(fetch),
+            formatTiming(count),
+            formatTiming(compiler),
+          ).toString()
+      } catch (error) {
+        console.error('Error building table to copy:', error)
+      }
+    }
+    copyToClipboard(clipboardText.trim())
+  }, [createdAt, request, result])
+
+  const icon = useMemo(() => {
+    return jitCompiled ? (
+      <IconWithTooltip
+        icon={<ZapIcon size="16px" />}
+        placement="top"
+        tooltip="JIT Compiled"
+      />
+    ) : (
+      <CheckmarkOutlineIcon size="18px" />
+    )
+  }, [jitCompiled])
+
   return (
-    <Wrapper>
-      <Timestamp createdAt={createdAt} />
-      {jitCompiled ? (
-        <IconWithTooltip
-          icon={<ZapIcon size="16px" />}
-          placement="top"
-          tooltip="JIT Compiled"
-        />
-      ) : (
-        <CheckmarkOutlineIcon size="18px" />
-      )}
-      <Content>{content}</Content>
-      <SideContent>{sideContent}</SideContent>
-    </Wrapper>
+    <PopperHover
```

Comment:
i think if tooltip is added, then it should only be shown when icon is hovered, not the entire row.

please note this little button at the bottom right, which allows to expand the notifications area. When expanded, the current tooltip UX is not good IMO.

<img width="356" alt="Screenshot 2023-05-03 at 14 49 00" src="https://user-images.githubusercontent.com/4284659/235907701-64ff0816-cc43-45ac-9a14-20cf808b2960.png">

Reply:
I've updated it such that the tooltip now shows to the right and is before the timestamp:

![Screenshot 2023-04-28 at 3 43 38 PM](https://user-images.githubusercontent.com/67945/236066665-dabbcfea-d627-4aa7-a8d2-ef992e15eec7.png)